recipes-app/iot2050-firmware-update/files/iot2050-firmware-update.tmpl

```diff
@@ -710,12 +737,12 @@ def main(argv):
             print("\n")
             for env in envlist:
                 print(env)
-            preserved_env_input = input(
+            preserved_env_input = the_input(args.quiet, 'y',
                 "\nWarning: Preserve the above env variables. Confirm(Y/n)? ")
             if not preserved_env_input == "n":
                 updater.update_uboot_env(envlist)
 
-        reboot_input = input(
+        reboot_input = the_input(args.quiet, 'n',
             "\nWarning: Completed. Please reboot the device. Reboot now(y/N)? ")
```

Comment:
Refine the commit message, now not `all` answers yes.

Reply:
`except reboot is 'n'` is telling this mesage.